### PR TITLE
be verbose if we're going to prompt

### PR DESCRIPTION
**Pull request by kevinushey**
Since the output we display might be controlled by `renv.verbose`.

### CODE REVIEW

**Review by aronatkins — APPROVED** [review]
Should the documentation for any of the `prompt` arguments or the `renv.verbose` option discuss their interactions?

**Comment by kevinushey** [comment]
Hmm... we don't actually document the use of `renv.verbose` anywhere. Maybe it deserves a mention in a vignette?

File: R/scope.R [hunk 1]
```diff
@@ -389,3 +389,12 @@ renv_scope_caution <- function(value) {
...
 }
+
+renv_scope_verbose_if <- function(value) {
+  if (value) {
+    renv_scope_options(
+      renv.verbose = value,
+      scope = parent.frame()
+    )
+  }
+}
```
Comment (hadley) [inline R/scope.R:400]:
```suggestion
renv_scope_verbose_if <- function(value, scope = parent.frame()) {
  if (value) {
    renv_scope_options(
      renv.verbose = TRUE,
      scope = scope
    )
  }
}
```

`value` -> `TRUE` makes the logic a bit clearer IMO and I think it's good practice to always have a` scope` argument.

**Review by hadley — APPROVED** [review]
Agree that it would be good to document this somewhere, but it's not clear where.

**Pull request merged**